example/src/main/java/io/netty/example/http/file/HttpStaticFileServerHandler.java

```diff
@@ -110,37 +110,42 @@
     public static final String HTTP_DATE_GMT_TIMEZONE = "GMT";
     public static final int HTTP_CACHE_SECONDS = 60;
 
+    // Note, that this approach for storing an instance of "current" request works
+    // only in case we don't need to support requests pipelining.
+    private FullHttpRequest request;
```

Comment:
why ? It should work also with pipelining atm.

Reply:
The `request` is reassigned on each `channelRead0`. If we read, let's say, 3 requests consistently and the exception has occurred when writing the 2nd response, we still rely on the properties of the last request. I'm not sure if it's possible to catch in this particular scenario as the processing of each request is pretty streamlined.

Reply:
got it... I think thats fine.

Reply:
Yeah, I think that works too. Do you think it's better to remove the comment to avoid confusion?